Makefile

```diff
@@ -9,21 +9,35 @@ DEFAULT_WORKSPACE_DIR = "./workspace"
 DEFAULT_MODEL = "gpt-4-0125-preview"
 CONFIG_FILE = config.toml
 
+UNAME_SYS = $(shell uname -s)
+# fallback to python3 if python is not available (e.g. on macOS)
+PYTHON_BIN = $(shell which python3 || which python)
```

Comment:
The comment says use python as default, and if that isn't available use python3
The code line does the opposite, it defaults to python3.

Also what should happen if neither is available?

Reply:
My question, I plan to use `python` by default. Perhaps I should consider adding a check for when both `python3` and `python` are not available.